test/jdk/javax/swing/JColorChooser/Test8152419.java

```diff
@@ -43,9 +43,18 @@
 import javax.swing.UIManager;
 import javax.swing.border.EmptyBorder;
 
+import jtreg.SkippedException;
```

Comment:
jtreg tag @library needs to be added for this

Reply:
Updated.